src/input_xml.F90

```diff
@@ -2868,6 +2873,87 @@ subroutine read_tallies_xml()
     if (run_mode == MODE_PLOTTING) return
 
     ! ==========================================================================
+    ! READ DATA FOR DERIVATIVES
+
+    ! Get pointer list to XML <derivative> nodes and allocate global array.
+    call get_node_list(doc, "derivative", node_deriv_list)
+    allocate(tally_derivs(get_list_size(node_deriv_list)))
+
+    ! Make sure this is not an MG run.
+    if (.not. run_CE .and. get_list_size(node_deriv_list) > 0) call &
+         fatal_error("Differential tallies not supported in multi-group mode")
```

Comment:
Seems a bit awkward to use an `&` here. Perhaps just make this a standard multi if statement